Reqnroll/EnvironmentAccess/BuildMetadata.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Reqnroll.EnvironmentAccess
+{
+    public class BuildMetadata
```

Comment:
Could this be a record? Could it have init properties instead of public setters?

I think immutable objects make things easier